tajo-core/src/main/java/org/apache/tajo/engine/planner/physical/DistinctGroupbySortAggregationExec.java

```diff
@@ -172,6 +168,13 @@ public void close() throws IOException {
 
   @Override
   public void init() throws IOException {
+    super.init();
+
+    if (aggregateExecs != null) {
```

Comment:
@jinossy I'm sorry.
I tested below code. and it causes NPE. 
```java
   List<String> lists = null;
    for(String a : lists) {
    }
```

I think null checking is needed. sorry to disturb you.